examples/with-mocha/test/registerModules.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path')
+
+require('@babel/register')({
+  configFile: path.normalize(`${__dirname}/../babel.config.js`),
+  ignore: ['/node_modules'],
+})
+
+// Required per https://testing-library.com/docs/react-testing-library/setup#using-without-jest
+require('jsdom-global/register')
+
+// Support fetch in a node env. Requires the localstorage polyfill
+require('isomorphic-fetch')
+require('localstorage-polyfill')
```

Comment:
Do we need the `localstorage-polyfill` if we use `whatwg-fetch` instead? 

I find it the less things our examples have, the more straightforward they are to follow.

Reply:
@kettanaito localstorage would still need to be mocked or polyfilled here using whatwg-fetch. I can just include a mock if you think it's more transparent than including this polyfill or a mock lib, just let me know :)